core/src/term/mod.rs

```diff
@@ -1111,6 +1111,57 @@ pub enum UnaryOp {
     Dualize(),
 }
 
+impl fmt::Display for UnaryOp {
```

Comment:
I considered panicking for operators without a `%...%` form but in the end I don't think it would help us a lot. On the other hand, the `Display` implementation could be useful for debugging at some point.